src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -114,13 +114,12 @@
             <template
               v-if="hasRoleDirector(orgPerson) || hasRoleProprietor(orgPerson) || hasRolePartner(orgPerson)"
             >
-              <p
-                v-if="IsSame(orgPerson.mailingAddress, orgPerson.deliveryAddress, ['id'])"
-                class="peoples-roles-delivery-address info-text"
-              >
-                Same as Mailing Address
-              </p>
-              <DeliveryAddress v-else class="peoples-roles-delivery-address" :address="orgPerson.deliveryAddress"/>
+              <template v-if="IsSame(orgPerson.mailingAddress, orgPerson.deliveryAddress, ['id'])">
+                <span class="peoples-roles-delivery-address info-text">Same as Mailing Address</span>
```

Comment:
Same as above: the paragraph element added some unwanted styling so I changed it to a span.